src/terra-dev-site/guide/Internationalization/AddingTranslations.b.guide.md

```diff
@@ -0,0 +1,13 @@
+# Adding Translations To Components
+
+When adding translations to a component, we recommend putting the translations in a directory named `translations`. The aggregate translations tool will look for translations in [the following directories](https://github.com/cerner/terra-toolkit/blob/master/scripts/aggregate-translations/defaultSearchPatterns.js) by default. The default search pattern will look for translations a maximum of three levels deep.
+
+Here is an example directory setup:
```

Comment:
We should call out translations much be for each locale specified for the app.

We also need to detail that each translation key should be namespaced and unique to prevent translation key collision from other dependencies

Reply:
Updated in this commit: https://github.com/cerner/terra-ui/pull/132/commits/cc0c8713331225234ca6d889d6cca8802fbc4d3c